code/game/objects/structures/crates_lockers/closets/wardrobe.dm

```diff
@@ -85,19 +85,15 @@
 	new /obj/item/clothing/shoes/black(src)
 	new /obj/item/clothing/suit/nun(src)
 	new /obj/item/clothing/head/nun_hood(src)
-	new /obj/item/clothing/suit/chaplain_hoodie(src)
-	new /obj/item/clothing/head/chaplain_hood(src)
 	new /obj/item/clothing/suit/holidaypriest(src)
 	new /obj/item/clothing/under/wedding/bride_white(src)
-	new /obj/item/storage/backpack/cultpack (src)
 	new /obj/item/storage/fancy/candle_box(src)
 	new /obj/item/storage/fancy/candle_box(src)
 	new /obj/item/deck/tarot(src)
 	new /obj/item/device/radio/headset/headset_service(src)
 	new /obj/item/spirit_board(src)
 	new /obj/item/mesmetron(src)
 	new /obj/item/toy/plushie/therapy(src)
-	new /obj/item/book/manual/psych(src)
```

Comment:
Why did this get removed?

Reply:
Because it serves no purpose now that traumas are gone.